.github/workflows/test.yml

```
@@ -67,3 +67,42 @@ jobs:
           FOLDER: D3DB
           SKIP_EMPTY_COMMITS: true
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  build-website:
+    name: Run D3 Build of website for github pages
+    environment:
+      name: d3db-gh-pages
+      url: https://github.com/TechWorksHub/ManySecured-D3DB/tree/gh-pages
```

This should be the GitHub Pages URL (at least, I'm assuming that it will appear here)

```suggestion
      url: https://TechWorksHub.github.io/ManySecured-D3DB
```

---

Nevermind, you already have this in `deploy-pages`, see https://github.com/TechWorksHub/ManySecured-D3DB/blob/eaecdccab2c612617df4d8f97c285fc9b8af6498/.github/workflows/test.yml#L102